namespace-provisioner/troubleshooting.hbs.md

```diff
@@ -0,0 +1,136 @@
+# Namespace Provisioner Troubleshooting
+
+## <a id="controller-logs"></a>Controller logs
+
+For getting the logs when using the controller to manage the `desire-namespaces`
+
+```bash
+kubectl -n tap-namespace-provisioning logs deployments/controller-manager
+```
+
+Use `-f` to follow the log output
+
+## <a id="carvel-kapp-application-error"></a>Kapp Application error
```

Comment:
the terms provisioner app/Kapp app/Carvel app are used interchangeable throughout - I think the term provisioner application should be used consistently? please comment?

Reply:
stick with provisioner application and link back to definition in about

Reply:
link carvel to https://carvel.dev/kapp-controller/docs/latest/app-overview/